demo/app.js

```diff
@@ -655,6 +657,7 @@ const Demo = React.createClass({
               Footer Content
             </div>
           )}
+          footerStyle={{ padding: '40px 20px' }}
```

Comment:
What's convention on handling inline css or putting it at the bottom and calling it?